addons/api/mirage/factories/managed-groups.js

```diff
@@ -0,0 +1,17 @@
+import factory from '../generated/factories/managed-groups';
+import permissions from '../helpers/permissions';
+
+const types = ['password', 'OIDC'];
+
+export default factory.extend({
+  id: (i) => `managed-groups-id-${i}`,
+  type: (i) => types[i % types.length],
+
+  authorized_actions: () =>
+    permissions.authorizedActionsFor('managed-groups') || [
```

Comment:
@DhariniJeeva I want to make sure: from the permissions and capabilities POV, I am missing something else?

Reply:
looks good to me. wondering if we need a list/create here?

Reply:
I just added `authorized_collection_actions` with create and list as I saw in other factory. Is this what you were meaning @DhariniJeeva ?

Reply:
yes, that's what I meant.